README.md

```diff
@@ -207,7 +207,8 @@ Please help us to grow this list of supported banks:
 * Handelsbank
 * Deutsche Bank
 * Sparkasse KölnBonn
-* Commerz Bank
+* Commerzbank
+* Sparkasse Mayen #HAA doesn´t work as expected, use instead e.HKD
```

Comment:
this actually depends on which order types are configured for your user, normally the banks are quite mean when it comes to give those to the users